src/linker/Linker/LinkContext.cs

```diff
@@ -496,6 +499,12 @@ public void LogMessage (MessageContainer message)
 				return;
 			}
 
+			if (message.Category == MessageCategory.Warning &&
+				message.Version > WarnVersion) {
```

Comment:
How does this work if the `message.Version` is `null` ... it always evaluates to `false`?, right? (so unspecified version means report always). I think we should add a comment here (it's not obvious that `message.Version` is nullable, and even with that it's not obvious what `<` does on nullables)

Reply:
Yes, it'll be `false` - I will add a comment.